arch/x86/configs/x86_64_moor_defconfig

```diff
@@ -499,7 +499,7 @@ CONFIG_PM_SLEEP=y
 CONFIG_PM_SLEEP_SMP=y
 # CONFIG_PM_AUTOSLEEP is not set
 CONFIG_PM_WAKELOCKS=y
-CONFIG_PM_WAKELOCKS_LIMIT=100
+CONFIG_PM_WAKELOCKS_LIMIT=10
```

Comment:
stock dies already, keeps killing background apps. Don't think this will be apt for stock.